pkg/web/api/v1/transaction.go

```diff
@@ -206,6 +208,10 @@ func (c *Transaction) Model() (model *models.Transaction, err error) {
 	return model, nil
 }
 
+func (c *Transaction) TitleStatus() string {
```

Comment:
The status on the info page looked a bit better in title case, so I created this function. However, if it's preferred to handle this another way, let me know.